package.json

```diff
@@ -94,5 +94,8 @@
   "private": true,
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "jest-environment-jsdom": "^24.0.0"
+  }
```

Comment:
This isn't the right fix. It should be applied to the packages that use jest. e.g., jest-enzyme. 

This would only fix it for development, not for the published packages.